src/wallet/init.cpp

```diff
@@ -230,7 +238,7 @@ bool VerifyWallets()
         }
 
         std::string strWarning;
-        bool dbV = CWalletDB::VerifyDatabaseFile(walletFile, GetDataDir().string(), strWarning, strError);
+        bool dbV = CWalletDB::VerifyDatabaseFile(walletFile, GetWalletDir().string(), strWarning, strError);
```

Comment:
Seems overkill to call GetWalletDir() so many times in this function, we could just call and store it once. But no, no need to fix in this PR, this is easier to review.